src/components/cylc/workflow/Toolbar.vue

```diff
@@ -285,6 +374,16 @@ export default {
     },
     isStopped () {
       this.expecting.stop = null
+    },
+    nWindow (newVal) {
+      // the user has requested to change the window size
+      this.setGraphWindow(newVal)
+    },
+    nEdgeDistance (newVal) {
+      // the scheduler has reported that the window size has changed
+      if (newVal !== undefined) {
```

Comment:
I think it's safer to guard against the value being `null` too
```suggestion
      if (newVal != undefined) {
```